apps/web/components/booking/AvailableTimes.tsx

```diff
@@ -89,6 +91,7 @@ const AvailableTimes: FC<AvailableTimesProps> = ({
                 date: slot.time.format(),
                 type: eventTypeId,
                 slug: eventTypeSlug,
+                ...(recurringCount && !rescheduleUid && { count: recurringCount }),
```

Comment:
Treat the booking and its available times as recurring only when a count exist and when it's not a rescheduling workflow